integr8ly/managed-service-broker
# change to code ready

## Motivation
add code ready service and icon for the code ready GA 
https://github.com/integr8ly/installation/pull/335

## Verification Steps

deploy quay.io/integreatly/managed-service-broker:INTLY-775 for the MSB
delete and recreate the cluster service broker 
ensure core ready shows up in the catalog
deploy to a namespace
Ensure you can reach the dashboard url
